Makefile

```diff
@@ -1,22 +1,59 @@
+ARCH    ?= amd64
+ALL_ARCH = amd64 arm arm64 ppc64le s390x
+
 .PHONY: all test
 
 IMAGE_BUILD_CMD := docker build
 
 VERSION := $(shell git describe --tags --dirty --always)
 
+IMAGEARCH ?=
+QEMUARCH ?= x86_64
+
+ifeq ($(ARCH),arm)
+        IMAGEARCH = arm32v7/
+        QEMUARCH = arm
+endif
+ifeq ($(ARCH),arm64)
+        IMAGEARCH = arm64v8/
+        QEMUARCH = aarch64
+endif
+ifeq ($(ARCH),ppc64le)
+        IMAGEARCH = ppc64le/
+        QEMUARCH = ppc64le
+endif
+ifeq ($(ARCH),s390x)
+        IMAGEARCH = s390x/
+        QEMUARCH = s390x
+endif
+
```

Comment:
Should we have a construct something like
```
ifeq
...
else ifeq
...
else ifeq
...
else
$(error unknown arch "$(ARCH)")
endif
```

Reply:
Done.

Reply:
Done.